### PR TITLE
Feature/kris nova logger

### LINKED ISSUE #325: Update Kubicorn Logger to kris-nova/logger
*Before creating a feature request, please search existing feature requests to see if you find a similar one. If there is a similar feature request please up-vote it and/or add your comments to it instead*

**Why do you want this feature?**
@kris-nova has broken the logger out to make it easier to consume, this will also allow us to turn off the timestamps making the logger feel more natural as a CLI logger

**What feature/behavior/change do you want?**
Update `kubicorn/logger` references to `kris-nova/logger`

### PR DESCRIPTION
**Pull request by christopherhein**
### Description
Changing logger to `kris-nova/logger` and using the new `Timestamps` option to turn off `Timestamps` by default. 

Also easter egg… adds `--fabulous` to use the Fabulous mode ;) @kris-nova

### Checklist
- [x] Code compiles correctly (i.e `make build`)
- [x] Added tests that cover your change (if possible)
- [x] All tests passing (i.e. `make test`)
- [x] Added/modified documentation as required (such as the README)
- [x] Added yourself to the `humans.txt` file

screen shot 2018-11-27 at 7 05 09 pm

I'm not a huge fan of having to set `-C false` to make `--fabulous` mode work, but I think for now it works.

Closes #325

### CODE REVIEW

**Comment by christopherhein** [comment]
@errordeveloper hope you are enjoying the vacay, when you get back would love your input :)\

File: cmd/eksctl/main.go [hunk 1]
```diff
@@ -24,11 +24,12 @@ var rootCmd = &cobra.Command{
 }
 
 func init() {
-
 	addCommands()
 
 	rootCmd.PersistentFlags().IntVarP(&logger.Level, "verbose", "v", 3, "set log level, use 0 to silence, 4 for debugging and 5 for debugging with AWS debug logging")
 	rootCmd.PersistentFlags().BoolVarP(&logger.Color, "color", "C", true, "toggle colorized logs")
```
Comment (errordeveloper) [inline cmd/eksctl/main.go]: I would rather change the `--color` flag to take a string, we can easily keep it compatible with current behaviour by supporting `--color=false`.

File: cmd/eksctl/main.go [hunk 2]
```diff
@@ -24,11 +24,12 @@ var rootCmd = &cobra.Command{
...
+	rootCmd.PersistentFlags().BoolVar(&logger.Timestamps, "timestamps", false, "toggle timestamps in logs")
+	rootCmd.PersistentFlags().BoolVar(&logger.Fabulous, "fabulous", false, "toggle fabulous log output")
```
Comment (errordeveloper) [inline cmd/eksctl/main.go]: I would rather change the `--color` flag to take a string, we can easily keep it compatible with current behaviour by supporting `--color=false`.

File: cmd/eksctl/main.go [hunk 3]
```diff
@@ -24,11 +24,12 @@ var rootCmd = &cobra.Command{
 }
 
 func init() {
-
 	addCommands()
 
 	rootCmd.PersistentFlags().IntVarP(&logger.Level, "verbose", "v", 3, "set log level, use 0 to silence, 4 for debugging and 5 for debugging with AWS debug logging")
 	rootCmd.PersistentFlags().BoolVarP(&logger.Color, "color", "C", true, "toggle colorized logs")
+	rootCmd.PersistentFlags().BoolVar(&logger.Timestamps, "timestamps", false, "toggle timestamps in logs")
```
Comment (errordeveloper) [inline cmd/eksctl/main.go]: I would avoid adding more flags. We can make this part `--verbose`, disable timestamps at the default level, and enable them in higher levels.

File: pkg/cfn/manager/nodegroup.go [hunk 4]
```diff
@@ -5,14 +5,13 @@ import (
 	"fmt"
 
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/pkg/errors"
-	"github.com/weaveworks/eksctl/pkg/cfn/builder"
-	"github.com/weaveworks/eksctl/pkg/eks/api"
-
 	cfn "github.com/aws/aws-sdk-go/service/cloudformation"
-	"github.com/kubicorn/kubicorn/pkg/logger"
+	"github.com/kris-nova/logger"
```
Comment (errordeveloper) [inline pkg/cfn/manager/nodegroup.go]: originally we tried to keep imports in logical groups, but clearly those have lost meaning over time and got messy... :(

Comment (christopherhein) [reply pkg/cfn/manager/nodegroup.go]: Sure, this is being done automatically by `goimports` and my IDE.

File: cmd/eksctl/main.go (hunk 3 above)
Comment (christopherhein) [reply cmd/eksctl/main.go]: What level would you expect this to be apart of?

File: cmd/eksctl/main.go (hunk 2 above)
Comment (christopherhein) [reply cmd/eksctl/main.go]: so the idea would be `--color=fabulous` or `--color=false`?

File: cmd/eksctl/main.go (hunk 3 above)
Comment (christopherhein) [reply cmd/eksctl/main.go]: Let me know if you want me to do this differently, I've implemented it like this:

```
logger.Timestamps = false
if logger.Level >= 4 {
  logger.Timestamps = true
}
```

**Comment by christopherhein** [comment]
Cool tests pass, let me know if you want anything done differently. @errordeveloper

**Pull request merged**